eslint.config.js

```diff
@@ -1,8 +1,16 @@
 import maninak from '@maninak/eslint-config'
 
-export default maninak({
-  ignores: ['*.md'],
-  typescript: {
-    tsconfigPath: 'tsconfig.json',
+export default maninak(
+  {
+    ignores: ['*.md'],
+    typescript: {
+      tsconfigPath: 'tsconfig.json',
+    },
   },
-})
+  {
+    files: ['**/*.test.ts'],
+    rules: {
+      'max-len': ['error', 120],
```

Comment:
Why do we need this?

Reply:
The name of the test I wrote was triggering the 95 default character limit.

Looked into the rule a bit more, and found out there's an `ignoreStrings` option, which covers this case.

There's also the `ignorePattern` option, which would let us be more specific (e.g. ignore lines starting with `it(`) but I feel `ignoreStrings` is good enough.

Unfortunately, I needed to re-iterate the whole rule and couldn't just update the `ignoreStrings` option.

Reply:
I haven't used the ignoreStrings on purpose. Too long strings in the source code should be broken down into multiple lines either with a `"" + ""` or as template strings

```js
const str = `multi\
line\
string`
```

In the case of tests I wouldn't mind too much if they were _sometimes_ longer e.g. 120 columns, but to be honest, since the code there is max 95 cols, it would still stand out and impede readability in narrower viewports.

Most importantly see [this inline comment](https://github.com/cytechmobile/radicle-planning-boards/pull/73#discussion_r1641789857) for more argumentation as to why I'm resisting this change (but I'm open to revisiting this decision later).

Reply:
Sounds good, reverted the eslint changes.